### ISSUE TITLE
Not all drives are properly mapped on Windows 10 Pro + WSL 

### ISSUE THREAD

**Issue by DenisSemerikov** [docksal/docksal#429]
**Description**

Windows 10 Pro + WSL + Docker for Windows always have virtual host = ".docksal"

**Steps to reproduce the issue:**

1.Fresh Windows Installation.
2. All software installed using this tutorial
https://blog.docksal.io/how-to-access-docker-for-windows-from-within-ubuntu-app-windows-services-for-linux-wsl-10ca8ef15b6e
3. fin project create

             -= Docksal project wizard =-

1. Enter the name of your project (no spaces or slashes): test-docksal

2. What would you like to install?
   [1] Drupal 7
   [2] Drupal 8
   [3] Wordpress
   [4] Magento
   [5] Static HTML site
   [6] Grav CMS
   [7] Gatsby JS
   [8] Laravel

Enter your choice (1-8): 5

Your site will be created at /d/Work/test-docksal
Your site will run Plain HTML
The URL of your site will be http://test-docksal.docksal

Do you wish to proceed? [y/n]: y
Starting services...
Creating stacks_db_1 ...
Creating stacks_cli_1 ...
Creating stacks_db_1
Creating stacks_cli_1 ... done
Creating stacks_web_1 ...
Creating stacks_web_1 ... done
Waiting 10s for container to start...
Virtual Host: .docksal
Done! Visit http://test-docksal.docksal
evildan@DESKTOP-K44LQNB:/mnt/d/Work$ fin pl
project             STATUS              virtual host          project root
stacks              Up 39 seconds       .docksal,*..docksal   /d/Work/test-docksal
evildan@DESKTOP-K44LQNB:/mnt/d/Work$ cd test-docksal/
evildan@DESKTOP-K44LQNB:/mnt/d/Work/test-docksal$ ls -la .docksal/
total 0
drwxrwxrwx 0 root root 4096 Dec 15 22:38 .
drwxrwxrwx 0 root root 4096 Dec 15 22:38 ..
evildan@DESKTOP-K44LQNB:/mnt/d/Work/test-docksal$ echo 'VIRTUAL_HOST=test.docksal' > .docksal/docksal.env
evildan@DESKTOP-K44LQNB:/mnt/d/Work/test-docksal$ cat .docksal/docksal.env
VIRTUAL_HOST=test.docksal

**Describe the results you received:**

http://test-docksal.docksal/ - Project Not Found.

**Describe the results you expected:**

Working site

**Output of `fin config`:**

<details>
  <summary>fin config output</summary>

```
---------------------
COMPOSE_PROJECT_NAME_SAFE:
COMPOSE_FILE:
/home/evildan/.docksal/stacks/volumes-bind.yml
/home/evildan/.docksal/stacks/stack-default.yml

PROJECT_ROOT: /d/Work/test-docksal
DOCROOT: docroot
VIRTUAL_HOST: .docksal
VIRTUAL_HOST_ALIASES: *..docksal
IP: 192.168.64.100
MYSQL: 192.168.64.100:32772

Docker Compose configuration
---------------------
services:
  cli:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      DOCROOT: docroot
      HOST_GID: '1000'
      HOST_UID: '1000'
      XDEBUG_CONFIG: remote_connect_back=0 remote_host=192.168.64.1
      XDEBUG_ENABLED: '0'
    hostname: cli
    image: docksal/cli:1.3-php7
    volumes:
    - docksal_ssh_agent:/.ssh-agent:ro
    - /home/docker
    - project_root:/var/www:rw,nocopy
  db:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      MYSQL_DATABASE: default
      MYSQL_PASSWORD: user
      MYSQL_ROOT_PASSWORD: root
      MYSQL_USER: user
    hostname: db
    image: docksal/db:1.1-mysql-5.6
    ports:
    - 3306/tcp
    volumes:
    - project_root:/var/www:ro,nocopy
  web:
    depends_on:
      cli:
        condition: service_started
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      APACHE_BASIC_AUTH_PASS: null
      APACHE_BASIC_AUTH_USER: null
      APACHE_DOCUMENTROOT: /var/www/docroot
    hostname: web
    image: docksal/web:2.1-apache2.4
    labels:
      io.docksal.project-root: /d/Work/test-docksal
      io.docksal.virtual-host: .docksal,*..docksal
    volumes:
    - project_root:/var/www:ro,nocopy
version: '2.1'
volumes:
  docksal_ssh_agent:
    external: true
    name: docksal_ssh_agent
  project_root:
    driver: local
    driver_opts:
      device: /d/Work/test-docksal
      o: bind
      type: none

---------------------
```

</details>

<br>

**Output of `fin sysinfo`:**

<details>
  <summary>fin sysinfo output</summary>

  ```
 fin sysinfo
███  OS
Linux DESKTOP-K44LQNB 4.4.0-43-Microsoft #1-Microsoft Wed Dec 31 14:42:53 PST 2014 x86_64 x86_64 x86_64 GNU/Linux
Mode: VirtualBox VM

███  FIN
fin version: 1.43.2

███  ENVIRONMENT
DOCKER_HOST             | 0.0.0.0:2375
 WARNING:  Non-default DOCKER_HOST

███  DOCKER
EXPECTED VERSION: 17.09.0-ce

Client:
 Version:      17.09.1-ce
 API version:  1.32
 Go version:   go1.8.3
 Git commit:   19e2cf6
 Built:        Thu Dec  7 22:24:23 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.09.1-ce
 API version:  1.32 (minimum version 1.12)
 Go version:   go1.8.3
 Git commit:   19e2cf6
 Built:        Thu Dec  7 22:28:28 2017
 OS/Arch:      linux/amd64
 Experimental: true

███  DOCKER INFO
Containers: 6
 Running: 6
 Paused: 0
 Stopped: 0
Images: 7
Server Version: 17.09.1-ce
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host ipvlan macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 06b9cb35161009dcb7123345749fef02f7cea8e0
runc version: 3f2f8b84a77f73d38244dd690525642a72156c64
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.9.49-moby
Operating System: Alpine Linux v3.5
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 1.934GiB
Name: moby
ID: NLUS:RT6T:CMZN:PTVU:AKH2:MHME:ZBBK:XI7C:7NT3:QHWB:YFFP:ZM5Z
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 63
 Goroutines: 75
 System Time: 2017-12-15T20:52:34.494878Z
 EventsListeners: 2
Registry: https://index.docker.io/v1/
Experimental: true
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

███  DOCKER COMPOSE
EXPECTED VERSION: 1.17.1
docker-compose version 1.17.1, build 6d101fb
docker-py version: 2.5.1
CPython version: 2.7.13
OpenSSL version: OpenSSL 1.0.1t  3 May 2016

███  DOCKER: IMAGES
REPOSITORY            TAG                 IMAGE ID            CREATED             SIZE
docksal/cli           1.3-php7            19bdddfeab89        2 months ago        759MB
docksal/web           2.1-apache2.4       32c1d32eba6d        3 months ago        87.5MB
docksal/vhost-proxy   1.1                 898fb21e7f2d        4 months ago        158MB
docksal/db            1.1-mysql-5.6       6feb79af9597        4 months ago        298MB
docksal/dns           1.0                 20c7a535479d        10 months ago       61.6MB
docksal/ssh-agent     1.0                 148220cc033d        10 months ago       12.2MB
d4w/nsenter           latest              9e4f13a0901e        14 months ago       83.8kB

███  DOCKER: CONTAINERS
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS                    PORTS                                                    NAMES
33a0368fd120        docksal/web:2.1-apache2.4   "httpd-foreground"       13 minutes ago      Up 13 minutes             80/tcp, 443/tcp                                          stacks_web_1
1bf61d83855c        docksal/db:1.1-mysql-5.6    "/entrypoint.sh my..."   13 minutes ago      Up 13 minutes             0.0.0.0:32772->3306/tcp                                  stacks_db_1
fe673903fbd0        docksal/cli:1.3-php7        "/opt/startup.sh s..."   13 minutes ago      Up 13 minutes (healthy)   22/tcp, 9000/tcp                                         stacks_cli_1
2a8f2b2a8030        docksal/ssh-agent:1.0       "/run.sh ssh-agent"      14 minutes ago      Up 14 minutes                                                                      docksal-ssh-agent
80ff29f9da62        docksal/dns:1.0             "/opt/entrypoint.s..."   14 minutes ago      Up 14 minutes             192.168.64.100:53->53/udp                                docksal-dns
49c6c4a1d131        docksal/vhost-proxy:1.1     "docker-entrypoint..."   14 minutes ago      Up 14 minutes             192.168.64.100:80->80/tcp, 192.168.64.100:443->443/tcp   docksal-vhost-proxy

███  HDD Usage
docker-machine executable was not found. (Try running 'fin update')
  ```

</details>

**Comment by achekulaev** [comment]
Hey man. Can't replicate the issue on my VmWare at all, the only difference is that I have no disk `D:`.
Could you try doing the same but on disk `C:`? 

```
cd /mnt/c/Work
fin project create
```

And tell if it still fails.

**Comment by DenisSemerikov** [comment]
Yep, it works perfect on disk c:
Bug or feature?

**Comment by DenisSemerikov** [comment]
now i see that only c: bind mounting to /c. In babun there are all drives.
manual bind mount or even symbolic link solves this problem.

**Comment by achekulaev** [comment]
Gotcha. I guess I see where the error is now.

**Comment by DenisSemerikov** [comment]
btw... c: & e: is ok for me

/mnt/c/Windows/System32$ fin reset system
Creating bind mount /mnt/c -> /c
[sudo] password for evildan:
Creating bind mount /mnt/e -> /e
Resetting network settings...

**Comment by achekulaev** [comment]
maybe d is already mapped? can you show output of `mount`?

**Comment by lmakarov** [comment]
@DenisSemerikov ping any updates?

**Comment by DenisSemerikov** [comment]
Yesterday I did "fin update". And today just after reboot I started evaluated bash.
`evildan@DESKTOP-K44LQNB:/mnt/c/Windows/System32$ ls -la /
total 52
drwxr-xr-x  0 root root  4096 Jan 10 08:42 .
drwxr-xr-x  0 root root  4096 Jan 10 08:42 ..
drwxr-xr-x  0 root root  4096 Sep 22 19:15 bin
drwxr-xr-x  0 root root  4096 Sep 22 19:15 boot
drwxr-xr-x  0 root root  4096 Jan 10 07:36 c
drwxr-xr-x  0 root root  4096 Jan 10 08:42 d
drwxr-xr-x  0 root root  4096 Jan 13 20:27 dev
drwxr-xr-x  0 root root  4096 Jan 10 07:36 e
drwxr-xr-x  0 root root  4096 Dec 22 18:06 etc
drwxr-xr-x  0 root root  4096 Dec 15 21:30 home
-rwxr-xr-x  1 root root 51272 Jan  1  1970 init
drwxr-xr-x  0 root root  4096 Sep 22 19:15 lib
drwxr-xr-x  0 root root  4096 Sep 22 19:13 lib64
drwxr-xr-x  0 root root  4096 Sep 22 19:12 media
drwxr-xr-x  0 root root  4096 Dec 22 20:38 mnt
drwxr-xr-x  0 root root  4096 Sep 22 19:12 opt
dr-xr-xr-x  8 root root     0 Jan 13 20:27 proc
drwx------  0 root root  4096 Dec 19 16:28 root
drwxr-xr-x  0 root root  4096 Jan 13 20:27 run
drwxr-xr-x  0 root root  4096 Dec 15 21:54 sbin
drwxr-xr-x  0 root root  4096 Aug 31 13:17 snap
drwxr-xr-x  0 root root  4096 Sep 22 19:12 srv
dr-xr-xr-x 12 root root     0 Jan 13 20:27 sys
drwxrwxrwt  0 root root  4096 Jan 10 10:19 tmp
drwxr-xr-x  0 root root  4096 Sep 22 19:12 usr
drwxr-xr-x  0 root root  4096 Sep 22 19:15 var
evildan@DESKTOP-K44LQNB:/mnt/c/Windows/System32$ mount
rootfs on / type lxfs (rw,noatime)
sysfs on /sys type sysfs (rw,nosuid,nodev,noexec,noatime)
proc on /proc type proc (rw,nosuid,nodev,noexec,noatime)
none on /dev type tmpfs (rw,noatime,mode=755)
devpts on /dev/pts type devpts (rw,nosuid,noexec,noatime)
none on /run type tmpfs (rw,nosuid,noexec,noatime,mode=755)
none on /run/lock type tmpfs (rw,nosuid,nodev,noexec,noatime)
none on /run/shm type tmpfs (rw,nosuid,nodev,noatime)
none on /run/user type tmpfs (rw,nosuid,nodev,noexec,noatime,mode=755)
binfmt_misc on /proc/sys/fs/binfmt_misc type binfmt_misc (rw,noatime)
C: on /mnt/c type drvfs (rw,noatime)
D: on /mnt/d type drvfs (rw,noatime)
evildan@DESKTOP-K44LQNB:/mnt/c/Windows/System32$ sudo rm /c
[sudo] password for evildan:
rm: cannot remove '/c': Is a directory
evildan@DESKTOP-K44LQNB:/mnt/c/Windows/System32$ sudo rmdir /c
evildan@DESKTOP-K44LQNB:/mnt/c/Windows/System32$ sudo rmdir /d
evildan@DESKTOP-K44LQNB:/mnt/c/Windows/System32$ sudo rmdir /e
evildan@DESKTOP-K44LQNB:/mnt/c/Windows/System32$ ls -la /
total 52
drwxr-xr-x  0 root root  4096 Jan 13 20:28 .
drwxr-xr-x  0 root root  4096 Jan 13 20:28 ..
drwxr-xr-x  0 root root  4096 Sep 22 19:15 bin
drwxr-xr-x  0 root root  4096 Sep 22 19:15 boot
drwxr-xr-x  0 root root  4096 Jan 13 20:27 dev
drwxr-xr-x  0 root root  4096 Dec 22 18:06 etc
drwxr-xr-x  0 root root  4096 Dec 15 21:30 home
-rwxr-xr-x  1 root root 51272 Jan  1  1970 init
drwxr-xr-x  0 root root  4096 Sep 22 19:15 lib
drwxr-xr-x  0 root root  4096 Sep 22 19:13 lib64
drwxr-xr-x  0 root root  4096 Sep 22 19:12 media
drwxr-xr-x  0 root root  4096 Dec 22 20:38 mnt
drwxr-xr-x  0 root root  4096 Sep 22 19:12 opt
dr-xr-xr-x  8 root root     0 Jan 13 20:27 proc
drwx------  0 root root  4096 Dec 19 16:28 root
drwxr-xr-x  0 root root  4096 Jan 13 20:28 run
drwxr-xr-x  0 root root  4096 Dec 15 21:54 sbin
drwxr-xr-x  0 root root  4096 Aug 31 13:17 snap
drwxr-xr-x  0 root root  4096 Sep 22 19:12 srv
dr-xr-xr-x 12 root root     0 Jan 13 20:27 sys
drwxrwxrwt  0 root root  4096 Jan 10 10:19 tmp
drwxr-xr-x  0 root root  4096 Sep 22 19:12 usr
drwxr-xr-x  0 root root  4096 Sep 22 19:15 var
evildan@DESKTOP-K44LQNB:/mnt/c/Windows/System32$ fin reset system
Creating bind mount /mnt/c -> /c
Creating bind mount /mnt/e -> /e
Resetting network settings...
The object already exists.


The object already exists.


Resetting Docksal services...
 * proxy
 * dns and resolver for .docksal domain
Enabling automatic *.docksal DNS resolver...

Ok.

Clearing DNS cache...
 * ssh-agent
Identity added: id_rsa (id_rsa)
evildan@DESKTOP-K44LQNB:/mnt/c/Windows/System32$ mount
rootfs on / type lxfs (rw,noatime)
sysfs on /sys type sysfs (rw,nosuid,nodev,noexec,noatime)
proc on /proc type proc (rw,nosuid,nodev,noexec,noatime)
none on /dev type tmpfs (rw,noatime,mode=755)
devpts on /dev/pts type devpts (rw,nosuid,noexec,noatime)
none on /run type tmpfs (rw,nosuid,noexec,noatime,mode=755)
none on /run/lock type tmpfs (rw,nosuid,nodev,noexec,noatime)
none on /run/shm type tmpfs (rw,nosuid,nodev,noatime)
none on /run/user type tmpfs (rw,nosuid,nodev,noexec,noatime,mode=755)
binfmt_misc on /proc/sys/fs/binfmt_misc type binfmt_misc (rw,noatime)
C: on /mnt/c type drvfs (rw,noatime)
D: on /mnt/d type drvfs (rw,noatime)
C: on /c type drvfs (rw,noatime)
rootfs on /e type lxfs (rw,noatime)`

**Comment by shelane** [comment]
Related work #890

**Comment by achekulaev** [comment]
It is not fixed there yet, but yeah a good point. I need to change looping `/mnt` for all drives to looping `wmic logicaldisk get drivetype,name` just like it is in `docker_machine_mount_smb`

**Comment by lmakarov** [comment]
Fixed in #989
